ownCloud/Server List/ServerListTableHeaderView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ServerListTableHeaderView.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 27.01.19.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+
+class ServerListTableHeaderView: UIView, Themeable {
+
+    var messageThemeApplierToken : ThemeApplierToken?
+    var textLabel : UILabel = UILabel()
+    
+    
+    override init(frame: CGRect) {
+        
+        super.init(frame: frame)
+        
+        Theme.shared.register(client: self, applyImmediately: true)
+        
+        textLabel.translatesAutoresizingMaskIntoConstraints = false
+        textLabel.text = "Locations".localized
+        textLabel.font = UIFont.boldSystemFont(ofSize: 24.0)
+        
+        self.addSubview(textLabel)
+        
+        let shadowView = UIView()
+        shadowView.backgroundColor = UIColor.init(white: 0.0, alpha: 0.1)
```

Comment:
@hosy This is jut one pixel line at the bottom? Just was thinking if it can use a color which is the result of blending the background color of the parent view with 10% opaque black to avoid compositing all-together.

Reply:
I don´t understand what you mean. This should looks like the shadow line of a bar from UINavigationBarController.

Reply:
@hosy I was just thinking that using solid color in the correct shade would make rendering faster. But here the benefit is probably neglectable.